core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -872,6 +882,33 @@ protected void doRun() throws Exception {
     }
 
     /**
+     * Get the Jenkins {@link StartupType startup type}.   
+     * @return The Jenkins {@link StartupType startup type}.
+     * @since FIXME
+     */
+    public StartupType getStartupType() {
+        return startupType;
+    }
+
+    /**
+     * Get the URL path to the Install Wizard JavaScript.
+     * @return The URL path to the Install Wizard JavaScript.
+     * @since FIXME
+     */
+    public String getInstallWizardJS() {
+        return servletContext.getInitParameter("install-wizard-js");
+    }
+
+    /**
+     * Save the last execute version file.
+     * @since FIXME
+     */
+    public void doSaveLastExecVersion() {
```

Comment:
Actually ... I think I might be able to get rid of this altogether, so don't bother justifying your comment. Thanks.